src/server/run-server.ts

```diff
@@ -1,14 +1,79 @@
 /**
  * Responsible for running the application is server.ts file
  */
-
+import { getConnection } from 'typeorm';
 import app from './server';
 import logger from '../logger';
-import { startConsumingEnrichment } from '../api/kafka';
+import { Hermeticity } from '../infrastructure/entitys/hermeticity';
+import { createConnection } from 'typeorm';
+import { MPPHermeticity, HermeticityStatus, hermeticityType } from '../core/hermeticity';
+import { MPPAlert, Severity, alertType } from '../core/alert';
+import { Alert } from '../infrastructure/entitys/alert';
+import { AllEnrichmentResponse, EnrichmentRepo } from '../core/repository';
+
+const enrichmentRepo: EnrichmentRepo = {
+  async addHermeticity(Hermeticity: MPPHermeticity): Promise<void> {
+    createConnection()
+      .then(async connection => {
+        return connection.manager.save(Hermeticity).then(examp => {
+          console.log('Hermeticity has been saved. Hermeticity id is', Hermeticity.ID);
+        });
+      })
+      .catch(error => console.log(error));
+  },
+  async addAlert(Alert: MPPAlert): Promise<void> {
+    createConnection()
+      .then(async connection => {
+        return connection.manager.save(Alert).then(examp => {
+          console.log('Alert has been saved. Alert id is', Alert.ID);
+        });
+      })
+      .catch(error => console.log(error));
+  },
+  async getAllEnrichment(): Promise<AllEnrichmentResponse> {
+    createConnection()
+      .then(async connection => {
+        const savedHermeticity = await getConnection().query('EXEC SelectAllActiveHermeticity', [5086]);
+        const savedAlert = await getConnection().query('EXEC SelectAllActiveALert', [5086]);
+        //const savedEnrichment: AllEnrichmentResponse = {["alert"]:savedAlert, ["hermeticity"]:savedHermeticity}
+        return { ['alert']: savedAlert, ['hermeticity']: savedHermeticity } as AllEnrichmentResponse;
+      })
+      .catch(error => console.log(error));
+  }
```

Comment:
Why is it defined here and not in the infrastructure file?
Dont use `then` , `catch`, change to `async`, `try...catch`.
Removed `console.log` statements, if needed use the logger instead.
The function `getAllEnrichment` does not return anything.
We need to decide how we want to deal with exceptions that happen in the db side, did you check what happens when the db is not available?

Reply:
Why is it defined here and not in the infrastructure file?
לא הבנתי את השאלה ...
מבחינת אם הDB לא זמין,
זה מחזיר שגיאה אבל אין מנגנון ששמור את המידע איפשהוא,
אבל בכללי DB זה דבר מאוד יציב, 
אז לדעתי לא צריך להשתגע בלהכין מנגנון כזה, מעיקר שהמגיע שלנו תזמי, אז לא מאוד אכפת לנו מאיבוד מידע